src/components/LoginView.vue

```diff
@@ -94,28 +101,34 @@ export default {
 };
 </script>
 
+<!-- TODO: the login/register view have the exact same style section
+should combine for no repetition -->
```

Comment:
I agree, please follow these instructions to eliminate duplication: https://stackoverflow.com/questions/48806433/sharing-common-css-across-vuejs-components